contribs/gmf/src/directives/authenticationdirective.js

```diff
@@ -182,19 +192,18 @@ gmf.AuthenticationController = function(gettextCatalog, $scope,
    */
   this.newPwdConfVal = '';
 
+  ol.events.listen(gmfAuthentication, gmf.AuthenticationEventType.READY,
+    this.onLoginReady_.bind(this));
 };
 
 
 /**
  * Initialise the controller.
  */
 gmf.AuthenticationController.prototype.$onInit = function() {
-  if (this.allowPasswordReset === undefined) {
-    this.allowPasswordReset = true;
-  }
-  if (this.allowPasswordChange === undefined) {
-    this.allowPasswordChange = true;
-  }
+  this.allowPasswordReset = !(this.allowPasswordReset === false);
```

Comment:
=> this.allowPasswordReset !== false
not?

Reply:
yes